concordium-consensus/src/Concordium/KonsensusV1/TreeState/StartUp.hs

```diff
@@ -186,7 +187,7 @@ loadSkovData _runtimeParameters = do
     -- TODO: When the database storage is modified to allow this, load the block. Issue #843
     let _finalizingCertifiedBlock = Absent
     _skovEpochBakers <- makeEpochBakers lastFinBlock
-
+    finBlockSeedstate <- getSeedState $ bpState lastFinBlock
```

Comment:
```suggestion
    finBlockSeedState <- getSeedState $ bpState lastFinBlock
```

Reply:
Applied suggestion in the latest commit.